src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
As requested by the previous Code Reviewer please consider resolving the following error from your tests to meet the project requirements.

## More details about ERROR:-

![image](https://user-images.githubusercontent.com/94223367/231873924-775c88cb-2ce6-4774-b4ca-009d07bca921.png)